articles/install-guide/qjupyter.md

```diff
@@ -65,4 +65,6 @@ IQ# (pronounced i-q-sharp) is an extension primarily used by Jupyter and Python
 
 ## Next steps
 
-Now that you have installed the Quantum Development Kit in your preferred environment, you can write and run [your first quantum program](xref:microsoft.quantum.quickstarts.qrng).
+- You can take a look to the [IQ\# code sample](https://docs.microsoft.com/en-us/samples/microsoft/quantum/intro-to-qsharp-jupyter/). There you will find a Q# Jupyter Notebook that shows how to use Q# in this environment.
```

Comment:
```suggestion
- You can take a look at the [IQ\# code sample](https://docs.microsoft.com/en-us/samples/microsoft/quantum/intro-to-qsharp-jupyter/). There you will find a Q# Jupyter Notebook that shows how to use Q# in this environment.
```
Just in general "take a look at" sounds more natural in this form. "Look to" works more in a context of "You can look to ... for ..."

Reply:
Thanks! Finding the right prepositions is one of the most difficult things for non-native speakers because there's no rule at all lol